extensions/amp-tabs/0.1/amp-tabs.js

```diff
@@ -25,15 +26,79 @@ export class AmpTabs extends AMP.BaseElement {
   /** @param {!AmpElement} element */
   constructor(element) {
     super(element);
+
+    /** @private {?NodeList} */
+    this.sections_ = null;
   }
 
   /** @override */
   buildCallback() {
+    //TODO: handle case where there are > 1 <amp-tabs> on a page.
+
     user().assert(isExperimentOn(this.win, TAG),
         `Experiment "${TAG}" is disabled.`);
+
+    let selectedTabHeaderIndex = -1;
+    let selectedTabContentIndex = -1;
+    this.sections_ = this.getRealChildren();
+    this.sections_.forEach((section, index) => {
+      /** The first section should be the tab headers */
+      if (!index) {
+        user().assert(
+            section.tagName.toLowerCase() == 'ul',
+            'Tab headers should be enclosed in a <ul> tag, ' +
+            'See https://github.com/ampproject/amphtml/blob/master/extensions/' +
+            'amp-tabs/amp-tabs.md. Found in: %s', this.element);
+        for (let i = 0; i < section.children.length; ++i) {
+          const tab = section.children[i];
+          tab.addEventListener('click',
+            this.onHeaderClick_.bind(this, i + 1 /* tab content index */));
+
+          if (tab.hasAttribute('selected')) {
+            selectedTabHeaderIndex = i;
+          }
+        }
+      } else {
+        /** The tab content */
+        user().assert(
+            section.tagName.toLowerCase() == 'section',
+            'Sections should be enclosed in a <section> tag, ' +
+            'See https://github.com/ampproject/amphtml/blob/master/extensions/' +
+            'amp-tabs/amp-tabs.md. Found in: %s', this.element);
+
+        if (section.hasAttribute('selected')) {
+          selectedTabContentIndex = index - 1;
+        }
+      }
+    });
+
+    user().assert(selectedTabHeaderIndex >= 0, 'You need to specify which ' +
+      'Tab Header will be selected by default. Add the "selected" attribute ' +
+      'to the <li> tag.');
+    user().assert(selectedTabContentIndex >= 0, 'You need to specify which ' +
+      'Tab Content will be displayed by default. Add the "selected" ' +
+      'attribute to the <section> tag.');
+    user().assert(selectedTabHeaderIndex == selectedTabContentIndex,
+      'Selected Tab Header and Selected Tab content indices do not match!');
+  }
+
+  /**
+   * Handles tab headers clicks to expand/collapse its content.
+   * @param {number} index the tab index to display
+   * @private
+   */
+  onHeaderClick_(index) {
+    event.preventDefault();
```

Comment:
I don't believe this is necessary. @camelburrito Do you know why accordion prevents default on tap of the header? There is no default behavior for click on `li` to prevent AFAIK.